zerver/lib/import_realm.py

```diff
@@ -1484,6 +1476,17 @@ def do_import_realm(import_dir: Path, subdomain: str, processes: int = 1) -> Rea
             realm_emoji.author_id = first_user_profile.id
             realm_emoji.save(update_fields=["author_id"])
 
+    channels = Stream.objects.filter(realm=realm)
+    for channel in channels:
+        # render_stream_description requires RealmEmoji in order to work correctly,
+        # so we had to ensure to import emojis first before calling it.
+        channel.rendered_description = render_stream_description(channel.description, realm)
+        # UserProfiles have been loaded, so now we're ready to set .creator_id
+        # for streams based on the mapping we saved earlier.
```

Comment:
oh, i guess while we're here we can modernize this comment to say `for channels based ....` instead of `streams`?